test/cases/regression-composer-works-behind-satellite-fallback.sh

```diff
@@ -65,14 +65,8 @@ case "${ID}" in
                 echo "Running on RHEL ${VERSION_ID}"
                 # starting in 8.5 the override file contains minor version number as well
                 VERSION_SUFFIX=$(echo "${VERSION_ID}" | tr -d ".")
-                if grep beta /etc/os-release;
-                then
-                    DISTRO_NAME="rhel-8-beta"
-                    REPOSITORY_OVERRIDE="/etc/osbuild-composer/repositories/rhel-${VERSION_SUFFIX}-beta.json"
-                else
-                    DISTRO_NAME="rhel-8"
-                    REPOSITORY_OVERRIDE="/etc/osbuild-composer/repositories/rhel-${VERSION_SUFFIX}.json"
-                fi
+                DISTRO_NAME="rhel-8"
```

Comment:
I'm not sure what is the intention here, but this will make the test case to use RHEL-8.3 repositories be copied as `$REPOSITORY_OVERRIDE`.